content/en/docs/concepts/containers/images.md

```diff
@@ -362,7 +276,7 @@ Kubernetes supports specifying container image registry keys on a Pod.
 
 #### Creating a Secret with a Docker config
```

Comment:
Do we also want to keep this example here? or point to the pulling images from a registry page as well? https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials

Reply:
Hmm. https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials doesn't exactly match https://kubernetes.io/docs/concepts/configuration/secret/#docker-config-secrets (different Secret types).

I suggest we omit that fix from _this_ PR and open a new, separate issue about harmonising the docs for this task. The “Docker” element here is more about either having `docker` on your PR or using Docker Hub to store images&mdash;the changes would be applicable to any container runtime that can pull images from a docker-style image repository.
That new issue should be prioritised important-soon, IMO.

Reply:
Ok, @sftim i can leave in this example. I can open a new issues, but I'm not sure i understand fully what the recommended action should be for the new issue you are proposing. Can you explain a little more?

Reply:
If you tag me on that new issue @a-mccarthy then I will try to check it and make sure it matches what I'd hoped to explain.